src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCardView/HomeStatisticsCardView.xib

```diff
@@ -185,7 +185,7 @@
                 <constraint firstItem="vUN-kp-3ea" firstAttribute="trailing" secondItem="Vvk-0F-7Ew" secondAttribute="trailing" constant="5" id="0Mw-LG-nd8"/>
                 <constraint firstItem="vUN-kp-3ea" firstAttribute="bottom" secondItem="Vvk-0F-7Ew" secondAttribute="bottom" id="Dha-Te-TYD"/>
                 <constraint firstItem="Vvk-0F-7Ew" firstAttribute="top" secondItem="vUN-kp-3ea" secondAttribute="top" id="Mug-VL-s2O"/>
-                <constraint firstItem="Vvk-0F-7Ew" firstAttribute="leading" secondItem="vUN-kp-3ea" secondAttribute="leading" constant="4" id="xdF-cA-CWy"/>
+                <constraint firstItem="Vvk-0F-7Ew" firstAttribute="leading" secondItem="vUN-kp-3ea" secondAttribute="leading" constant="16" id="xdF-cA-CWy"/>
```

Comment:
That doesn't also quite look right, now every this is shifted to the right. It think it would just make sense to keep the smaller spacing so the cells are properly left-aligned with the rest of the Home Screen and can still be seen when scrolled out to the left. Figma seems very inconsistent to me here